### PR TITLE
Addressing review comments of issue #83

### LINKED ISSUE #83: Header - Iteration 2
Goal: Refine the header that has been created in https://github.com/urfuwo/hlx-test/issues/75

Changes:

- When you resize the window to something between 980px and 1030px, then the icons on the rights (search, language switch) are cut-off. That should be changed

- When you resize the header to mobile view, the remaining elements are not vertically aligned (centered). Instead, the SAP logo appears to be too high / too close to the top, while the burger icon and the other icons appear to be too close to the bottom. I believe all should be vertically aligned

- When you resize the header to mobile view, the site label ("News Center") should not disappear, but remain, see https://www.figma.com/file/OFEfvhyXJvYJ6MQZ30699m/v2.0-Brand-Digital-Design-System?type=design&node-id=2822-829&mode=design&t=yNwqt7UTEFznnIXP-0

- The header-wrapper and the header (block) divs have different heights (64px vs 60px). I believe they should have the same height to make the next element (the hero) "bump up" against it and make the header entries look correctly vertically centered within the header element. Or else they look slightly "too high" (because they are centered in the header, but not centered in the header-wrapper)

General point of discussion:
- Does it help to make the structure of the navigation - at the editorial level - more uniform by converting all sections into bullet lists (ul/li). This way, the navigation structures would not requires dedicated CSS for p as well as for ul/li

References:
- Page: https://www.figma.com/file/oSetT4LbatRmXlcB2A7V8Z/Content-Hubs-2024?type=design&node-id=269-30889&mode=design&t=p98jyS19OoS4hi9y-0
- Header: https://www.figma.com/file/OFEfvhyXJvYJ6MQZ30699m/v2.0-Brand-Digital-Design-System?type=design&node-id=9601-13063&mode=design&t=66k43R4twXlQ0foN-0

### PR DESCRIPTION
**Pull request by saurabh-khare**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #83 

Test URLs:
- Before: https://main--hlx-test--urfuwo.hlx.live/blog/2023/11/supply-chain-issues-keeping-ceos-up-at-night
- After: https://83-header-iteration-2--hlx-test--urfuwo.hlx.page/draft/skhare/best-time-to-be-a-developer-is-now

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -1,13 +1,12 @@
 /* header and nav layout */
```
Comment (sdmcraft) [inline blocks/header/header.css:1]: 1. Looking at the [Frontify specs](https://sap.frontify.com/document/223143#/-/masthead), it seems a blue underline needs to come for the selected item.

image

2. Also the background color should be light grey.
3. The top/bottom paddings looks a little bit less than specs.
4. The left/right margins of the header look different from the page. Probably they should align though I didn't find that spelled out in the specs.

Comment (saurabh-khare) [reply blocks/header/header.css:1]: @sdmcraft Thanks for noticing the colour. I changed the background and pushed changes. 
For active section marker we need the site structure, so navigation know which page is active then make the active section underline blue. Hence not been implemented now.

Comment (saurabh-khare) [reply blocks/header/header.css:1]: Paddings and margins are referred from figma:
https://www.figma.com/file/OFEfvhyXJvYJ6MQZ30699m/v2.0-Brand-Digital-Design-System?type=design&node-id=8969%3A77791&mode=dev

Comment (sdmcraft) [reply blocks/header/header.css:1]: ok thanks. On point 4 above I think we need to align header with rest of the page but can be taken up separately.

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**